packages/blog/data/2021/2021-01-04-phpstan-abstract-parent-generics-dummies.md

```diff
@@ -25,7 +25,7 @@ The documentation [barely scratches this topic](https://phpstan.org/blog/generic
 
 <br>
 
-What is the use case? We have two classes - an abstract repository:
+What is the use case? We have two classes – an abstract repository:
```

Comment:
Handling this per item does not make sense and adds burgen to maintainer and incosistency.
So it would be useful to have CI check and fix to cover this.

Reply:
Sorry, I didn't learn anything about CI since https://github.com/symplify/symplify/pull/3498 ;-)
But I doubt that doing this automatically will ever work. Sometimes the short dash is the right character, sometimes the long dash ;-)

Reply:
If you can do it, so can code :)

Reply:
My best guess: If there's a blank before and after, it should be a long dash. But I'm sure that this isn't true in all situations. So I won't come up with an algo ;-)

Reply:
I must not fear. Fear is the mind killer.